--- a/src/Main.hs
+++ b/src/Main.hs
@@ -10,9 +10,9 @@ config =
         [ Run $ Memory ["-t", "Mem: <used> Gb", "--", "--scale", "1024"] 20
         , Run $ Kbd []
         , Run $ Date "%a %d/%m/%y <fc=#ee9a00>%H:%M</fc>" "date" 10
-        , Run $ XMonadLog
+        , Run $ Battery [ "-t", "<left>%"] 10
         ]
-    , template = "%XMonadLog% }{ %kbd% | %date% | %memory%"
+    , template = "}{ %kbd% | %date% | %memory% | %battery%"
     , alignSep = "}{"
     }
 
